site/includes/components/social-share.njk

```diff
@@ -0,0 +1,16 @@
+<div class="pb-8 flex flex-col md:flex-row mx-auto md:justify-center">
+    <div class="py-2 md:px-2 flex justify-center">
+            <a href="http://twitter.com/share?text=URGENT: NHS workers can’t work if they’re hungry. There are doctors in your local hospital who will be in the middle of their shift tonight and won&#39;t have anything to eat. Buy daily meals for @NHSuk staff, direct from local restaurants, at&url=http://MealsForTheNHS.com&hashtags=FeedTheNHS" target="_blank" class="mx-2 bg-blue-600 hover:bg-blue-700 transition-all ease-in-out duration-300 py-4 px-6 rounded-lg cursor-pointer text-white text-lg md:text-2xl flex justify-center items-center font-regular border-b-4 border-blue-900">
```

Comment:
Hey! This reads nicely but I've seen a lot of people sharing our headline too. I suggest:

After 18-hour shifts trying to save our lives from COVID-19, NHS Hospital Staff are faced with canteens that shut at 5pm. Help @feedthenhs get food to wards, today. &url=http://MealsForTheNHS.com&hashtags=FeedTheNHS